vm/systemSmartContracts/staking_test.go

```diff
@@ -1251,7 +1264,14 @@ func TestStakingSc_ExecuteStakeStakeJailAndSwitchWithBoundaries(t *testing.T) {
 			eei.blockChainHook = blockChainHook
 			args := createStakingSCArgs(eei, stakingAccessAddress, stakeValue, maxStakedNodesNumber)
 			enableEpochsHandler, _ := args.EnableEpochsHandler.(*enableEpochsHandlerMock.EnableEpochsHandlerStub)
-			enableEpochsHandler.IsCorrectJailedNotUnStakedEmptyQueueFlagEnabledInEpochCalled = tt.flagJailedRemoveEnabled
+			enableEpochsHandler.IsFlagEnabledCalled = func(flag core.EnableEpochFlag) bool {
```

Comment:
please use another implementation of the enable epoch stub to change only the corresponding flags

Reply:
changed